xExtension-CustomCSS/README.md

```diff
@@ -46,4 +46,40 @@ Desktop screen resolution:
 
 Mobile screen resolution:
 
-![Mobile](mobile_resolution.png)                          
+![Mobile](mobile_resolution.png)  
+
+#### Getting rid of Top Menu Items
+
+The Top Menu within the mobile view might look a little bit cluttered, depending on the theme. The following CSS rules allow to hide unneccessary top menu buttons or input boxes.
+```css
+    #nav_menu_actions /* Hides "Actions" Menu in Mobile View */
+    {
+        display:none;
+    }
```

Comment:
There are three separate suggestions here:
1. Put the comment first and above for clarity.
2. Keep FreshRSS style:
    * Put the opening brace on the same line as the selector.
    * Put a space in between the property and its value.
3. Don't indent everything for no apparent reason.

```suggestion
/* Hides "Actions" Menu in Mobile View */
#nav_menu_actions {
    display: none;
}
```

Reply:
Sorry, I am not a programmer, so I am very bad with coding. Also, this is my second time doing something with git, this is all very new to me. Hope I did everything right now ;)